docs/sources/installation/mac.md

```diff
@@ -9,49 +9,50 @@ weight = 500
 +++
 
 
-# Installing on macOS
+# Install on macOS
 
-This page provides instructions to help you install Grafana on macOS. You can either install using homebrew or a binary tar file.
+This page provides instructions to help you install Grafana on macOS. You can either install using Homebrew or a binary tar file.
 
 
-## Install using homebrew
+## Install with Homebrew
 
-Installation can be done using [homebrew](http://brew.sh/)
+Installation can be done using [homebrew](http://brew.sh/).
 
-Install latest stable:
+Install latest stable version:
 
 ```bash
 brew update
 brew install grafana
 ```
 
-To start grafana look at the command printed after the homebrew install completes.
-
-To upgrade use the reinstall command
+You can also install the latest unstable Grafana version from Git:
```

Comment:
Do we really want to tell them how to do this? Should this be in the Developer's Guide instead?

Reply:
@marcusolsson ?

Reply:
Any mention of building Grafana from source should link to the Developer Guide, but in this case I don't think we need to tell them how to install Grafana from master using brew at all.

Reply:
Removed all references to building Grafana from master in this doc, moved configuration information to configuration.md.